### PR TITLE
Update odometry fusion documentation

### PR DESCRIPTION
**Pull request by marcelino-pensa**
See issue https://github.com/PX4/Firmware/issues/15168

### CODE REVIEW

**Comment by hamishwillee** [comment]
Looks great to me @TSC21 , but you're the ROS expert.

**Comment by hamishwillee** [comment]
@TSC21 Please review.

File: en/ros/external_position_estimation.md [hunk 1]
```diff
@@ -152,17 +152,19 @@ MAVROS has plugins to relay a visual estimation from a VIO or MoCap system using
...
-/mavros/odometry/odom | [ODOMETRY](https://mavlink.io/en/messages/common.html#ODOMETRY) (`frame_id =` [MAV_FRAME_LOCAL_FRD](https://mavlink.io/en/messages/common.html#MAV_FRAME_LOCAL_FRD)) | `vehicle_visual_odometry`
+/mavros/odometry/out (`frame_id = odom`, `child_frame_id = base_link`) | [ODOMETRY](https://mavlink.io/en/messages/common.html#ODOMETRY) (`frame_id =` [MAV_FRAME_LOCAL_FRD](https://mavlink.io/en/messages/common.html#MAV_FRAME_LOCAL_FRD)) | `vehicle_visual_odometry`
 /mavros/mocap/pose | [ATT_POS_MOCAP](https://mavlink.io/en/messages/common.html#ATT_POS_MOCAP) | `vehicle_mocap_odometry`
-/mavros/odometry/odom | [ODOMETRY](https://mavlink.io/en/messages/common.html#ODOMETRY) (`frame_id =` [MAV_FRAME_MOCAP_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_MOCAP_NED)) | `vehicle_mocap_odometry`
+/mavros/odometry/out (`frame_id = odom`, `child_frame_id = base_link`) | [ODOMETRY](https://mavlink.io/en/messages/common.html#ODOMETRY) (`frame_id =` [MAV_FRAME_MOCAP_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_MOCAP_NED)) | `vehicle_mocap_odometry`
```
Comment (TSC21) [inline en/ros/external_position_estimation.md]:
```suggestion
/mavros/odometry/out (`frame_id = odom`, `child_frame_id = base_link`) | [ODOMETRY](https://mavlink.io/en/messages/common.html#ODOMETRY) (`frame_id =` [MAV_FRAME_LOCAL_FRD](https://mavlink.io/en/messages/common.html#MAV_FRAME_LOCAL_FRD)) | `vehicle_mocap_odometry`
```

**Review by TSC21 — CHANGES_REQUESTED** [review]
Just a small fix required and all good.

**Comment by hamishwillee** [comment]
@marcelino-pensa FYI I have merged this with @TSC21 correction (hope that OK with you). Thanks very much for taking the time to contribute!

**Comment by marcelino-pensa** [comment]
Thanks @hamishwillee! I was trying to hold this back until I had my questions answered in https://github.com/PX4/Firmware/issues/15168... I can update the documentation in the future if I get an answer for that question

**Comment by hamishwillee** [comment]
Cool. I tend to merge what is known as early as I can - everyone is busy, and you can hold up a lot of good info waiting on answers.

**Pull request merged**